Automattic/wp-calypso
# Create core web vitals display area

* Closes https://github.com/Automattic/dotcom-forge/issues/8561
* Closes https://github.com/Automattic/dotcom-forge/issues/8566
* Closes https://github.com/Automattic/dotcom-forge/issues/8567

## Proposed Changes

* create the components displaying the core web vitals and their explanations

## Testing Instructions



- Visit `/speed-test-tool?url=http%3A%2F%2Fexample.com`
- Check the rendered metrics against the Iix1FFHIQdfECIXn5JKT5s-fi-4356_19644
- Click around on the different tabs, and check that:
  - the metrics are displayed with the correct units
  - the explanation copy is correct
  - the scale showing the score is correct
- You can change the props of the `CoreWebVitalsDisplay` component to alter the metrics shown

| Active metric | Preview |
|--------|--------|
| FCP | ![image](https://github.com/user-attachments/assets/424ee8cf-ee16-4ab6-a36d-1547326550c9) |
| LCP | ![image](https://github.com/user-attachments/assets/c8253f03-616f-4866-937c-e85a2b4e4a07) |
| CLS | ![image](https://github.com/user-attachments/assets/dea69bda-aaea-48af-864f-177149720588) |
| INP | ![image](https://github.com/user-attachments/assets/1781f704-a113-4ce7-ba66-ff5ed002f863) |
| TTFB | ![image](https://github.com/user-attachments/assets/1faca572-c22a-4d9b-a1e9-c67183441e8b) | 

## Known issues/differences

- The legend are rendered at the thresholds rather than as a linear scale. This change made the implementation much simpler

| Figma | Implementation |
|--------|--------|
| <img width="400" alt="image" src="https://github.com/user-attachments/assets/2ec0b6e9-730f-4bd0-8d6c-c4c1f3c159e7"> | <img width="534" alt="image" src="https://github.com/user-attachments/assets/fcc12191-c652-4de7-92ef-fb01a439179c"> |

## Pre-merge Checklist
